packages/client/src/stores/builder.ts

```diff
@@ -2,9 +2,35 @@ import { writable, get } from "svelte/store"
 import { API } from "@/api"
 import { devToolsStore } from "./devTools.js"
 import { eventStore } from "./events.js"
+import {
+  ComponentDefinition,
+  DropPosition,
+  PingSource,
+  PreviewDevice,
+  Screen,
+} from "@budibase/types"
+
+interface BuilderStore {
+  inBuilder: boolean
+  screen: Screen | null
+  selectedComponentId: string | null
+  editMode: boolean
+  previewId: string | null
+  theme: string | null
+  customTheme: string | null
+  previewDevice: PreviewDevice
+  navigation: string | null
+  hiddenComponentIds: []
+  usedPlugins: { name: string; hash: string }[] | null
+  eventResolvers: {}
+  metadata: { componentId: string; step: number } | null
```

Comment:
I can't find any usages of this metadata... maybe we can remove it too?

Reply:
This is used in `MultiStepFormblock.svelte`